strider/fetcher.py

```diff
@@ -1,14 +1,4 @@
-"""Fetcher 2.0.
-
- .oooooo..o     .             o8o        .o8
```

Comment:
RIP asci art!

Reply:
No! It was just moved to the server.py file!